### DISCUSSION TITLE
Using `next/link` to external page?

### DISCUSSION THREAD

**Discussion by crs1138** [vercel/next.js#54095] [Help] [answered]
### Summary

For linking to an external page, what is the current (v13+) recommendation? Using the `<Link />` component or the HTML `<a>` element?

If there is no preference, why doesn't `<Link />` have the `target` prop for enabling to open links in new window?

**Comment by icyJoseph** [comment]
In Next 13, the Link does have a target prop. https://nextjs.org/docs/app/api-reference/components/link#props

> Good to know: <a> tag attributes such as className or target="_blank" can be added to <Link> as props and will be passed to the underlying <a> element.

**Reply by icyJoseph** [reply, marked as answer]
Though, to answer your question, I'd personally use a plain `a` tag for statically known externals links.

Calculating at runtime whether a dynamic string is an external link or not, is bit of overhead. The framework will do the right thing, and even use the target prop, etc.

**Reply by crs1138** [reply]
Thanks, I missed that. In our project we are using NextJS as a generator of static pages thus this overhead should not be an issue.
